src/lib/sql/sql_pipeline.hpp

```diff
@@ -14,6 +14,16 @@
 
 namespace opossum {
 
+// Holds relevant information about the execution of an SQLPipeline.
+struct SQLPipelineMetrics {
+  std::vector<std::reference_wrapper<const SQLPipelineStatementMetrics>> statement_metrics;
+
+  // This is different from the other measured times as we only get this for all statements at once
+  std::chrono::microseconds parse_time_micros{};
```

Comment:
I *think* this doesn't initialise the duration to zero, which is probably what you intended http://en.cppreference.com/w/cpp/chrono/duration/duration